assets/stylesheets/responsive/custom.scss

```diff
@@ -458,21 +458,16 @@ a.link_button_green_large {
 
 div.correspondence {
   background-color: $correspondence-bg;
-  padding-bottom: 0.5em;
-  .event_actions {
-    margin-bottom: 0;
```

Comment:
Yeah that is a bit rubbish. Um... "Update CSS to use the new request page layout" would probably be better (I'll check - and split if necessary - then update)